code/modules/reagents/chemistry/recipes/drugs.dm

```diff
@@ -37,3 +36,21 @@
 /datum/chemical_reaction/pumpup
 	results = list(/datum/reagent/drug/pumpup = 5)
 	required_reagents = list(/datum/reagent/medicine/epinephrine = 2, /datum/reagent/consumable/coffee = 5)
+
+/datum/chemical_reaction/maint_tar1
+	results = list(/datum/reagent/toxin/acid = 1 ,/datum/reagent/drug/maint/tar = 3)
+	required_reagents = list(/datum/reagent/consumable/tea = 1, /datum/reagent/yuck = 1 , /datum/reagent/fuel = 1)
+
+/datum/chemical_reaction/maint_tar2
+	results = list(/datum/reagent/toxin/acid = 1 ,/datum/reagent/drug/maint/tar = 3)
+	required_reagents = list(/datum/reagent/consumable/tea = 1, /datum/reagent/consumable/enzyme = 3 , /datum/reagent/fuel = 1)
```

Comment:
enzyme is kind of a rare/limited chem (outside of the kitchen's roundstart supply and broborgs), dude

you sure you don't wanna make it, like, a catalyst instead of a consumed reactant?

Reply:
enzyime can be mass made with biogen which meta has a public one